src/sentry/api/endpoints/organization_code_mappings.py

```diff
@@ -134,7 +134,7 @@ def get(self, request: Request, organization) -> Response:
         integration_id = request.GET.get("integrationId")
         project_id = request.GET.get("projectId")
 
-        if not integration_id and not project_id:
+        if not (integration_id or project_id):
```

Comment:
This check is no longer valid. Passing no `project_id` means "all projects I am a member of".